torch/csrc/generic/Storage.cpp

```diff
@@ -7,7 +7,7 @@ PyObject *THPStorageClass = nullptr;
 PyObject * THPStorage_(New)(c10::intrusive_ptr<c10::StorageImpl> ptr)
 {
   AT_ASSERT(ptr);
-  PyTypeObject *type = (PyTypeObject *)THPStorageClass;
+  PyTypeObject *type = (PyTypeObject *)THPByteStorageClass;
```

Comment:
We should probably rename THPByteStorageClass

Reply:
Although, instead of changing `THPByteStorageClass` to `THPStorageClass`, maybe there's a better name to indicate that it's a handle for the Python class `torch._UntypedStorage`. Maybe `THPUntypedStorage`? However, I guess one of the long term goals is to probably remove typed storages, isn't it? So maybe `THPStorageClass` is fine

Reply:
UntypedStorage is good as intermediate. But if we want to big step to the final state we can leave it be until we're done.